dist/centrifuge.d.ts

```diff
@@ -0,0 +1,116 @@
+// Type definitions for centrifuge 2.1.5
```

Comment:
I think we can omit version here as it won't be automatically updated?

Reply:
Maybe:
// Type definitions for centrifuge 2.*.*

Reply:
Yeah, `2.X` or `2.X.X` will be fine I suppose